contracts/RewardPool.vy

```diff
@@ -144,14 +142,14 @@ def _find_timestamp_epoch(ve: address, _timestamp: uint256) -> uint256:
 
 @view
 @internal
-def _find_timestamp_user_epoch(ve: address, user: address, _timestamp: uint256, max_user_epoch: uint256) -> uint256:
+def _find_timestamp_user_epoch(user: address, _timestamp: uint256, max_user_epoch: uint256) -> uint256:
```

Comment:
i think these utilities could be exposed from veyfi? that would also be cheaper and less code in this contract.